src/app/public/modules/flyout/flyout-iterator.component.ts

```diff
@@ -26,18 +39,32 @@ export class SkyFlyoutIteratorComponent implements OnDestroy {
     return this._previousButtonClick;
   }
 
-  private _previousButtonClick = new EventEmitter<void>();
-
   @Output()
   public get nextButtonClick(): EventEmitter<void> {
     return this._nextButtonClick;
   }
 
+  public themeName: string;
+
+  private ngUnsubscribe = new Subject();
+
   private _nextButtonClick = new EventEmitter<void>();
 
-  constructor() {}
+  private _previousButtonClick = new EventEmitter<void>();
+
+  constructor(
+    public themeSvc: SkyThemeService
```

Comment:
You can make this `private`, or better yet, just remove the access modifier altogether.
```
constructor(
  themeSvc: SkyThemeService
) { }
```

Reply:
Done.